go.mod

```diff
@@ -23,12 +24,14 @@ require (
 	github.com/spf13/cobra v0.0.3
 	github.com/spf13/pflag v1.0.2 // indirect
 	golang.org/x/crypto v0.0.0-20180910181607-0e37d006457b
+	golang.org/x/oauth2 v0.0.0-20190402181905-9f3314589c9a // indirect
+	golang.org/x/time v0.0.0-20190308202827-9d24e82272b4 // indirect
 	gopkg.in/airbrake/gobrake.v2 v2.0.9 // indirect
 	gopkg.in/gemnasium/logrus-airbrake-hook.v2 v2.1.2 // indirect
 	gopkg.in/inf.v0 v0.9.1 // indirect
 	k8s.io/api v0.0.0-20181026145037-6e4b5aa967ee // indirect
 	k8s.io/apimachinery v0.0.0-20181126191516-4a9a8137c0a1
-	k8s.io/client-go v7.0.0+incompatible
+	k8s.io/client-go v10.0.0+incompatible
```

Comment:
I'd expect that `k8s.io/apimachinery`, `k8s.io/api` and potentially even `k8s.io/klog` are bumped in the `go.sum` file after this change? It seems like `go mod tidy` (or `hack/update-vendor.sh`) needs running 😄